tensorflow/core/kernels/data/experimental/lmdb_dataset_op_test.cc

```diff
@@ -0,0 +1,283 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "tensorflow/core/kernels/data/experimental/lmdb_dataset_op.h"
+
+#include "tensorflow/core/kernels/data/dataset_test_base.h"
+#include "tensorflow/core/lib/io/path.h"
+#include "tensorflow/core/platform/env.h"
+#include "tensorflow/core/platform/test.h"
+
+namespace tensorflow {
+namespace data {
+namespace experimental {
+namespace {
+
+constexpr char kNodeName[] = "lmdb_dataset";
+constexpr char kIteratorPrefix[] = "Iterator";
+constexpr char kDataFileName[] = "data.mdb";
+constexpr char kDataFileLoc[] = "core/lib/lmdb/testdata";
+
+class LMDBDatasetParams : public DatasetParams {
+ public:
+  LMDBDatasetParams(std::vector<std::string> filenames,
+                    DataTypeVector output_dtypes,
+                    std::vector<PartialTensorShape> output_shapes,
+                    string node_name)
+      : DatasetParams(std::move(output_dtypes), std::move(output_shapes),
+                      std::move(kNodeName)),
+        filenames_(CreateTensor<std::string>(
+            TensorShape({static_cast<int>(filenames.size())}), filenames)) {}
+
+  virtual std::vector<Tensor> GetInputTensors() const override {
+    return {filenames_};
+  }
+
+  virtual Status GetInputNames(
+      std::vector<string>* input_names) const override {
+    *input_names = {LMDBDatasetOp::kFileNames};
+    return Status::OK();
+  }
+
+  virtual Status GetAttributes(AttributeVector* attributes) const override {
+    *attributes = {{LMDBDatasetOp::kOutputTypes, output_dtypes_},
+                   {LMDBDatasetOp::kOutputShapes, output_shapes_}};
+    return Status::OK();
+  }
+
+  virtual string dataset_type() const override {
+    return LMDBDatasetOp::kDatasetType;
+  }
+
+ private:
+  // Names of binary database files to read, boxed up inside a Tensor of
+  // strings
+  Tensor filenames_;
+};
+
+class LMDBDatasetOpTest : public DatasetOpsTestBaseV2 {};
```

Comment:
@frreiss As `DatasetOpsTestBaseV2` has been merged into `DatasetOpsTestBase`, could you please change `DatasetOpsTestBaseV2` to `DatasetOpsTestBase`?

Reply:
Changed the base class to `DatasetOpsTestBase` in commit ca5e8a2.